website_v2/gcloud-powershell-website-v2/www/static/css/style.css

```diff
@@ -1,27 +1,27 @@
 /* TODO: Bootstrap, hit this with the css hammer to make sure it's standard. */
-.TOC {
+.table-of-contents {
     /*light grey color*/
     background-color: #fafafa;
-    float:left;
+    float: left;
     height: 800px;
     margin-top: 10px;
-    overflow-y: auto;
-    position: fixed;
+    overflow: hidden;
+    height: 1%;
     width: 400px;
 }
-.Info {
+.informationArea {
```

Comment:
generally css classes are not camelcased and use '-'s.  Such as information-area.  Like you did with table-of-contents